packages/mdc-textfield/label/README.md

```diff
@@ -0,0 +1,50 @@
+<!--docs:
+title: "Text Field Label"
+layout: detail
+section: components
+excerpt: "The label is a text caption or description for the text field."
+iconId: text_field
+path: /catalog/input-controls/text-fields/label/
+-->
+
+# Text Field Label
+
+The label is a text caption or description for the text field.
+
+## Design & API Documentation
+
+<ul class="icon-list">
+  <li class="icon-list-item icon-list-item--spec">
+    <a href="https://material.io/guidelines/components/text-fields.html#text-fields-layout">Material Design guidelines: Text Fields Layout</a>
+  </li>
+</ul>
+
+
+## Usage
+
+#### MDCTextFieldLabel API
```

Comment:
These heading levels don't make sense to me... Usage is h2, this is h4, foundation is h5, but then "using the foundation class" is h3?

Now that I look, the same applies to #1697. Maybe we should have a task to iron out the subcomponents' readme formats once they all exist.

Reply:
Filed #1707.